config/file_config.go

```diff
@@ -51,7 +51,34 @@ type samplerConfigType struct {
 // NewConfig creates a new config struct
 func NewConfig(config, rules string) (Config, error) {
 	c := viper.New()
-	c.SetDefault("sendticker", 100*time.Millisecond)
+
+	var defaultPeers []string
+	defaultPeers = append(defaultPeers, "http://127.0.0.1:8081")
+
+	var defaultAPIKeys []string
+	defaultAPIKeys = append(defaultAPIKeys, "*")
+
+	c.SetDefault("ListenAddr", "0.0.0.0:8080")
+	c.SetDefault("PeerListenAddr", "0.0.0.0:8081")
+	c.SetDefault("APIKeys", defaultAPIKeys)
+	c.SetDefault("Peers", defaultPeers)
+	c.SetDefault("RedisHost", "localhost:6379")
```

Comment:
I think this probably shouldn't have a default as if you are using redis it will be in a cluster and you probably won't be using localhost